### ISSUE TITLE
Docs: Parameter 'reject' not supported for epochs in ica.fit()?

### ISSUE THREAD

**Issue by sab-d** [autoreject/autoreject#170]
Hi,

I was wondering if there's a mistake in the FAQ section of the documentation ("Should I apply ICA first or autoreject first?").
According to the MNE docs the `reject` parameter in `ica.fit()` only applies to an instance of Raw, not Epochs.
However, here `ica.fit()` is used with `reject` on epochs several times, e.g.:

    >>> reject = get_rejection_threshold(epochs)
    >>> ica.fit(epochs, reject=reject)

**Comment by jasmainak** [comment]
You are right, it looks like a typo and should be `raw` not `epochs` when you pass to `ica.fit`. Could you perhaps make a pull request to fix it?

**Comment by zuxfoucault** [comment]
For the same purpose from above, would it be legitimated to `epochs_clean = ar.transform(epochs)` then `ica.fit(epochs_clean)`? Thanks!

**Comment by jasmainak** [comment]
you can try but I'm guessing autoreject would drop/interpolate many trials because of eye-blinks. The procedure we outline in the FAQ is to avoid that situation.

**Comment by balandongiv** [comment]
> 
> 
> You are right, it looks like a typo and should be `raw` not `epochs` when you pass to `ica.fit`. Could you perhaps make a pull request to fix it?

It seems the typo in the FAQ has not been fixed. Should I PR for this issue? @jasmainak ?

**Comment by jasmainak** [comment]
@balandongiv if you have the time to actually update an example with the proposed workflow, that would be really great!

you can take this one for instance: http://autoreject.github.io/auto_examples/plot_visualize_bad_epochs.html#sphx-glr-auto-examples-plot-visualize-bad-epochs-py

and add ICA processing to it. Do you want to give it a shot?

**Comment by balandongiv** [comment]
@jasmainak . Yes, I am more than happy to contribute.

**Comment by jasmainak** [comment]
Fantastic, looking forward to your PR
